src/boot/efi/cpio.c

```diff
@@ -301,29 +301,6 @@ static EFI_STATUS pack_cpio_trailer(
         return EFI_SUCCESS;
 }
 
-static char16_t *get_dropin_dir(const EFI_DEVICE_PATH *file_path) {
```

Comment:
hmm, if you make this public, maybe call this get_extra_dir() to make clear it's the ".extra.d" dir... otherwise "dropin" dir is a bit too generic, could just mean the dir you drop UKIs into or so.

Reply:
renamed